src/components/camera.js

```diff
@@ -1,6 +1,7 @@
 var registerComponent = require('../core/component').registerComponent;
 var THREE = require('../lib/three');
 var utils = require('../utils/');
+var bind = utils.function.bind;
```

Comment:
We may or may not, but I think it's smart to prepare for the possible scenarios. If we don't ever add more functions, then it might look cluttered (subjectively, I think it's organized). If we do and we end up namespacing it later, we have to call a breaking change and components using the function (possibly third-party) have to be migrated.